app/code/Magento/Swatches/Test/Mftf/Test/AdminCreateTextSwatchTest.xml

```diff
@@ -78,24 +77,25 @@
         <click selector="{{AdminCreateProductConfigurationsPanel.next}}" stepKey="clickOnNextButton4"/>
         <click selector="{{AdminProductFormActionSection.saveButton}}" stepKey="clickOnSaveButton2"/>
         <click selector="{{AdminChooseAffectedAttributeSetPopup.confirm}}" stepKey="clickOnConfirmInPopup"/>
-
+        <magentoCLI stepKey="indexerReindex" command="indexer:reindex" />
```

Comment:
@engcom-Echo doing full reindex and flush cache will significantly slow down functional tests. Could you get rid of it?
Context https://github.com/magento/magento2/pull/26990

Reply:
I hope a do that. :)